src/subdomains/core/sell-crypto/process/buy-fiat.entity.ts

```diff
@@ -336,6 +380,26 @@ export class BuyFiat extends IEntity {
   get isLightningTransaction(): boolean {
     return this.cryptoInputBlockchain === Blockchain.LIGHTNING;
   }
+
+  get user(): User {
+    return this.sell.user;
+  }
+
+  get userData(): UserData {
+    return this.user.userData;
+  }
+
+  get route(): Sell {
+    return this.sell;
+  }
+
+  get target(): { address: string; asset: Fiat; trimmedReturnAddress: string } {
+    return {
+      address: this.sell.deposit?.address,
+      asset: this.sell.fiat,
+      trimmedReturnAddress: this.sell?.iban ? Util.blankStart(this.sell.iban) : null,
```

Comment:
Is this correct? Shouldn't the address be the IBAN? And trimmedReturnAddress the user address?